source/initHQ/fortify.sqf

```diff
@@ -1,4 +1,5 @@
 _centerPos = _this select 0;
+_hq = _this select 1;
```

Comment:
Yes you're correct. It takes whatever the second parameter being passed on is and saves it into the variable you put.